src/data/faq.json

```diff
@@ -158,9 +159,11 @@
-                                    "See also the blog article: <a href='../../blog/2022-04-19-cwa-2-21/' target='_blank' >Corona-Warn-App version 2.21 now allows test management for family members</a>"
+                                    "<ol><li>Read through the leaflet carefully, to make sure if the QR code really is to be scanned with the Corona-Warn-App, or if it is meant for another test infrastructure.</li><li>Only if you made sure that the QR code is to be scanned with the Corona-Warn-App and not for another system, proceed in the Corona-Warn-App.</li><li>You can scan each PCR-Test QR code only once with the Corona-Warn-App. After scanning, the QR code expires. This prevents misusing the QR code.</li><li>If you received a PCR-Test QR code for your own test, scan it with the Corona-Warn-App and wait until the result is retrieved. This can take from a couple of hours up to a few days. Open the Corona-Warn-App regularly to check if the test result is already available.</li><li>If you received additional QR codes for tests of your family members, for example, for your children who do not have their own phones, you can use them on your own device as well since version 2.21. There is no need to wait between scanning your own test results and those of family members. They are processed independently and their status can be viewed on the Status Tab.</li><li>If your QR code doesn't work and you received a positive test result: Call the TAN hotline (<a href='#international_phone_numbers' target='_blank' >click here to see the phone numbers</a>) to receive a teleTAN. You can then enter the teleTAN in your Corona-Warn-App so that the persons you encountered in the past days can be informed over the Corona-Warn-App. Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of ??????h. A warning can only be issued for a test result that is assigned to the owner of the device. You cannot warn on behalf of family members whose tests you have added to your device. Please pay attention to the recommendations the Corona-Warn-App displays.</li></ol>",

```

Comment:
In

"Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App."

"teleTAN" => "a teleTAN"